pimd/pim_util.h

```diff
@@ -25,4 +25,5 @@ int pim_is_group_224_4(struct in_addr group_addr);
 bool pim_is_group_filtered(struct pim_interface *pim_ifp, pim_addr *grp);
 int pim_get_all_mcast_group(struct prefix *prefix);
 bool pim_addr_is_multicast(pim_addr addr);
+bool is_invalid_ipddress(struct ipaddr ip);
```

Comment:
if this is going to be a public pim api, let's name it "pim_xxx"?
would it make sense to pass a 'const' pointer instead of the entire struct?
I'm a little on the fence about the sense of this test - should it be "valid" or "invalid" ? sometimes it's clearer for the sense of the test to be positive - it avoids the potential awkwardness of code that tries to test for "not-invalid".